src/bundle/Resources/views/content/content_on_the_fly/content_create_on_the_fly.html.twig

```diff
@@ -29,9 +29,11 @@
 
 {% block form_fields %}
     {{ parent() }}
-    {{ form_widget(form.publish, {'attr': {'hidden': 'hidden'}}) }}
-    {{ form_widget(form.saveDraft, {'attr': {'hidden': 'hidden'}}) }}
-    {{ form_widget(form.cancel, {'attr': {'hidden': 'hidden'}}) }}
+    {% if form.publish is defined %}{{ form_widget(form.publish, {'attr': {'hidden': 'hidden'}}) }}{% endif %}
+    {% if form.saveDraft is defined %}{{ form_widget(form.saveDraft, {'attr': {'hidden': 'hidden'}}) }}{% endif %}
+    {% if form.cancel is defined %}{% do form.cancel.setRendered() %}{% endif %}
```

Comment:
Why `cancel` should not be rendered? I think this one is used to discard draft?

Reply:
Yeah, `cancel` should be rendered. `saveDraft` button should not be a part of this form if you do what I proposed in https://github.com/ezsystems/ezplatform-admin-ui/pull/953#discussion_r274301870

Reply:
`cancel` is also used in `UserCreateType` and there is no option to disable it  by options